README.md

```diff
@@ -1121,6 +1121,7 @@ Table of Contents
 
 ## Other Free Resources
 
+  * [Bagisto](https://bagisto.com/en/) - An Opensource eCommerce platform for all to build and scale your business.
```

Comment:
This does not appear to be delivered as a SaaS? There's a live demo and download for self hosting, but no SaaS, this list is for SaaS.

Reply:
Hi,

I just added in the other free resources as per the heading.

Reply:
They are "other" but they are still services.  This is a list of services.

Reply:
So as per your requirement which topic is suitable.

Reply:
I can update as per the need.

Reply:
This whole file, all sections and topics is for services.  There is a few minor exceptions - IDEs mainly - otherwise this is for SaaS.

If your software is not delivered as a SaaS it cannot go on this list.

Reply:
OK got the point. So no listing for opensource ecomerce platforms.

Reply:
list-of-apples.com you are the pear growing being passive aggressive because his pear won’t be listed as a kind of apple. 

It’s not a slight or a aggression against open source. It’s just not the purpose of the list. Passive aggressive comments such as the one you left is uncalled for.